internal/resources/grafana/resource_alerting_rule_group_test.go

```diff
@@ -675,6 +675,34 @@ func TestAccAlertRule_NotificationSettings(t *testing.T) {
 	})
 }
 
+func TestAccRecordingRule(t *testing.T) {
+	testutils.CheckOSSTestsEnabled(t, ">=11.2.0")
```

Comment:
Can you switch that to run in a cloud instance instead as 11.2 isn't released yet

```suggestion
	testutils.CheckCloudInstanceTestsEnabled(t) # TODO: Use 11.2 when released
```